flow_process_components/FlowBaseComponents/force-app/main/default/lwc/fbc_pickObjectAndField/fbc_pickObjectAndField.js-meta.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
     <apiVersion>46.0</apiVersion>
-    <description>Pick Object And Field FSC</description>
+    <description>fbc_PickObjectAndField</description>
```

Comment:
-I0'd like to keep the descriptions and labels 'friendly'....  perhaps 'Pick Object and Field'[--m

Reply:
The other two that you exposed use the fbc_ label.  I was trying to be consistent.  Would you still like me to change this back?

Reply:
Yes, when you showed me the screen shot I realized that I had failed to put good labels in. So change them here and there as well, if you don't mind